<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorModeDialog</class>
 <widget class="QDialog" name="Mer::Internal::MerEmulatorModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Emulator Mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Emulator:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="deviceNameLabel">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Emulator used by the active project target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>none</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device &amp;model:</string>
       </property>
       <property name="buddy">
        <cstring>deviceModelComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="deviceModelComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="originalViewModeRadioButton">
       <property name="text">
        <string>&amp;Original</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">zoomButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="scaledViewModeRadioButton">
       <property name="text">
        <string>&amp;Scaled down (1:2)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">zoomButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="landscapeRadioButton">
       <property name="text">
        <string>&amp;Landscape</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">orientationButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="portraitRadioButton">
       <property name="text">
        <string>&amp;Portrait</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">orientationButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>View mode:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="restartEmulatorCheckBox">
       <property name="toolTip">
        <string>These changes take effect only at the next emulator start.</string>
       </property>
       <property name="text">
        <string>&amp;Restart emulator now</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>deviceModelComboBox</tabstop>
  <tabstop>portraitRadioButton</tabstop>
  <tabstop>landscapeRadioButton</tabstop>
  <tabstop>originalViewModeRadioButton</tabstop>
  <tabstop>scaledViewModeRadioButton</tabstop>
  <tabstop>restartEmulatorCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Mer::Internal::MerEmulatorModeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Mer::Internal::MerEmulatorModeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="orientationButtonGroup"/>
  <buttongroup name="zoomButtonGroup"/>
 </buttongroups>
</ui>
